### ISSUE TITLE
implement loading data from multiple datasources

### ISSUE THREAD

**Issue by jgrzebyta** [jgrzebyta/triple-loader#9]
Implement populating triple store with multiple data sources.

**Comment by jgrzebyta** [comment]
The unit test _test-load-multidata_ performs sometimes following error:

```

18:28:40,715 [clojure-agent-send-off-pool-0] ERROR sparql: There is error:  java.util.concurrent.ExecutionException
java.util.concurrent.ExecutionException: org.eclipse.rdf4j.repository.RepositoryException: org.eclipse.rdf4j.sail.SailException: org.apache.lucene.store.AlreadyClosedException: this IndexWriter is closed
    at java.util.concurrent.FutureTask.report(FutureTask.java:122)
    at java.util.concurrent.FutureTask.get(FutureTask.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at clojure.lang.Reflector.invokeMatchingMethod(Reflector.java:93)
    at clojure.lang.Reflector.invokeNoArgInstanceMember(Reflector.java:313)
    at sparql$load_multidata.invokeStatic(sparql.clj:110)
    at sparql$load_multidata.invoke(sparql.clj:90)
    at triple.multiload$fn__1108.invokeStatic(multiload.clj:15)
    at triple.multiload$fn__1108.invoke(multiload.clj:13)
    at clojure.test$test_var$fn__7983.invoke(test.clj:716)
    at clojure.test$test_var.invokeStatic(test.clj:716)
    at clojure.test$test_var.invoke(test.clj:707)
    at boot.user$eval133$fn__134.doInvoke(boot.user2229776919628937226.clj:21)
    at clojure.lang.RestFn.applyTo(RestFn.java:137)
    at clojure.lang.AFunction$1.doInvoke(AFunction.java:29)
    at clojure.lang.RestFn.applyTo(RestFn.java:137)
    at clojure.core$apply.invokeStatic(core.clj:646)
    at clojure.core$apply.invoke(core.clj:641)
    at boot.core$construct_tasks.doInvoke(core.clj:911)
    at clojure.lang.RestFn.applyTo(RestFn.java:137)
    at clojure.core$apply.invokeStatic(core.clj:646)
    at clojure.core$apply.invoke(core.clj:641)
    at boot.core$boot$fn__933.invoke(core.clj:949)
    at clojure.core$binding_conveyor_fn$fn__4676.invoke(core.clj:1938)
    at clojure.lang.AFn.call(AFn.java:18)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.rdf4j.repository.RepositoryException: org.eclipse.rdf4j.sail.SailException: org.apache.lucene.store.AlreadyClosedException: this IndexWriter is closed
    at org.eclipse.rdf4j.repository.sail.SailRepositoryConnection.commit(SailRepositoryConnection.java:172)
    at sparql$eval1086$fn__1087$fn__1088.invoke(sparql.clj:148)
    at sparql$eval1086$fn__1087.invoke(sparql.clj:130)
    at clojure.lang.MultiFn.invoke(MultiFn.java:238)
    at sparql$eval1082$fn__1083.invoke(sparql.clj:120)
    at clojure.lang.MultiFn.invoke(MultiFn.java:238)
    at sparql$load_multidata$fn__1062$fn__1063.invoke(sparql.clj:102)
    ... 5 more
Caused by: org.eclipse.rdf4j.sail.SailException: org.apache.lucene.store.AlreadyClosedException: this IndexWriter is closed
    at org.eclipse.rdf4j.sail.lucene.LuceneSailConnection.commit(LuceneSailConnection.java:237)
    at org.eclipse.rdf4j.repository.sail.SailRepositoryConnection.commit(SailRepositoryConnection.java:169)
    ... 11 more
Caused by: org.apache.lucene.store.AlreadyClosedException: this IndexWriter is closed
    at org.apache.lucene.index.IndexWriter.ensureOpen(IndexWriter.java:718)
    at org.apache.lucene.index.IndexWriter.commitInternal(IndexWriter.java:2856)
    at org.apache.lucene.index.IndexWriter.commit(IndexWriter.java:2833)
    at org.eclipse.rdf4j.sail.lucene.LuceneIndex.commit(LuceneIndex.java:649)
    at org.eclipse.rdf4j.sail.lucene.LuceneSailConnection.addRemoveStatements(LuceneSailConnection.java:252)
    at org.eclipse.rdf4j.sail.lucene.LuceneSailConnection.commit(LuceneSailConnection.java:217)
    ... 12 more
```

**Comment by jgrzebyta** [comment]
Fix in commit 4f6454bbce4d9571db575429ef1cf3a42b7dc88d .
